commentHandler.go

```diff
@@ -256,21 +263,72 @@ func manageLocking(req types.IssueCommentOuter, cmdType string) (string, error)
 	return buffer.String(), nil
 }
 
+func manageMilestone(req types.IssueCommentOuter, cmdType string, cmdValue string) (string, error) {
+
+	milestoneValue := cmdValue
+	var buffer bytes.Buffer
+	milestoneAction := strings.Replace(strings.ToLower(cmdType), "milestone", "", 1)
```

Comment:
I don't understand what this line is for.

Reply:
To print the action the right way else it will print the constant like
"mrtind wants to `SetMilestone` milestone of 'example' on example" 
and we want
"mrtind wants to `set` milestone of 'example' on example"